index.js

```diff
@@ -156,5 +164,7 @@ function gql(/* arguments */) {
 gql.default = gql;
 gql.resetCaches = resetCaches;
 gql.disableFragmentWarnings = disableFragmentWarnings;
+gql.print = require('graphql/language/printer');
+gql.parser = parser
```

Comment:
I was exporting the whole lib `var parser = require('graphql/language/parser');`